<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search</class>
 <widget class="QWidget" name="search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>-17</x>
      <y>0</y>
      <width>20</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>-17</x>
      <y>0</y>
      <width>20</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>-17</x>
      <y>0</y>
      <width>20</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>81</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-48</y>
      <width>118</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_15">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>51</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_16">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>20</width>
      <height>201</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_17">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>16</width>
      <height>201</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="face0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="song0">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="singer0">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="face1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="face2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="face3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="face4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="song1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="song2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="song3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="song4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="singer1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="singer2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="singer3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="singer4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image:none;
background-color:rgba(255,255,255,0);
border-radius:5px
}
QPushButton:hover
{
background-color:rgba(255,255,255,0.5);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/icons/chazhao.png</normaloff>:/new/prefix1/icons/chazhao.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/icons/bofang.png</normaloff>:/new/prefix1/icons/bofang.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/icons/bofang.png</normaloff>:/new/prefix1/icons/bofang.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/icons/bofang.png</normaloff>:/new/prefix1/icons/bofang.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/new/prefix1/icons/bofang.png</normaloff>:/new/prefix1/icons/bofang.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>41</width>
     <height>271</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image:none;
background-color:rgba(255,255,255,0);
border-radius:5px
}
QPushButton:hover
{
background-color:rgba(255,255,255,0.5);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/guanbi.png</normaloff>:/new/prefix1/icons/guanbi.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/bofang.png</normaloff>:/new/prefix1/icons/bofang.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
